.github/workflows/deploy-to-control-plane.yml

```
@@ -1,177 +1,193 @@
+          }
+          
+          # Get default branch
+          DEFAULT_BRANCH="refs/heads/$(gh repo view --json defaultBranchRef --jq '.defaultBranchRef.name')"
+          echo "DEFAULT_BRANCH=$DEFAULT_BRANCH" >> $GITHUB_OUTPUT
+          
+          # Check if we should deploy
+          if [[ "${{ github.ref }}" == "$DEFAULT_BRANCH" ]]; then
+            echo "SHOULD_DEPLOY=true" >> $GITHUB_OUTPUT
+            echo "IS_STAGING=true" >> $GITHUB_ENV
+            echo "APP_NAME=${{ env.STAGING_APP_NAME }}" >> $GITHUB_ENV
+          elif [[ "${{ github.event_name }}" == "issue_comment" && 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)